rspec-core.gemspec

```diff
@@ -44,7 +44,10 @@ Gem::Specification.new do |s|
     s.add_runtime_dependency "rspec-support", "~> #{RSpec::Core::Version::STRING.split('.')[0..1].concat(['3']).join('.')}"
   end
 
-  s.add_development_dependency "cucumber", "~> 1.3"
+  if RUBY_VERSION >= '2.5'
```

Comment:
Logic like this in the gemspec does not work since this will be evaluated at gem build time. This is probably going to be on a post-2.5 Ruby version, meaning the resulting gem will just build-depend on cucumber.